packages/data-provider/src/controllers/submittedDataController.ts

```diff
@@ -14,6 +14,44 @@ const controller = (dependencies: BaseDependencies) => {
 	const defaultPage = 1;
 	const defaultPageSize = 20;
 	return {
+		deleteSubmittedDataBySystemId: async (
+			req: Request<{ systemId: string }, {}, any, { dryRun: string; reason: string }>,
+			res: any,
+			next: any,
+		) => {
+			try {
+				const systemId = req.params.systemId;
+
+				// Gives false value except when query param is explicitly true
+				const dryRun = req.query.dryRun === 'true' ? true : false;
```

Comment:
because we could support TRUE, or True
```suggestion
				const dryRun = req.query.dryRun?.toLowerCase() === 'true' ? true : false;
```